Artifacts/windows-vsts-build-agent/vsts-agent-install.ps1

```diff
@@ -0,0 +1,122 @@
+# Downloads the Visual Studio Online Build Agent, installs on the new machine, registers with the Visual
+# Studio Online account, and adds to the specified build agent pool
+
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$true)]
+    $vstsAccount,
+    
+    [Parameter(Mandatory=$true)]
+    $vstsUserPassword,
+    
+    [Parameter(Mandatory=$true)]
+    $agentName,
+    
+    [Parameter(Mandatory=$true)]
+    $poolname,
+    
+    [Parameter(Mandatory=$true)]
+    $driveLetter
+)
+
```

Comment:
By default, this script will not stop execution upon the first error.  Set $ErrorActionPreference = "Stop" if you want to stop when the first error is thrown.